### ISSUE TITLE
v1.0.15 - AttributeError: 'NoneType' object has no attribute 'to_dict'

### ISSUE THREAD

**Issue by vk2him** [music-assistant/hass-music-assistant#128]
Got this this error  after a scan:

```
Logger: music_assistant.mass
Source: /usr/local/lib/python3.9/site-packages/music_assistant/mass.py:242
First occurred: 9:32:12 AM (1 occurrences)
Last logged: 9:32:12 AM

Job [Add filesystem://album/Fleetwood Mac to database] failed with error 'NoneType' object has no attribute 'to_dict'.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 70, in add
    db_item = await self.add_db_item(item)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 118, in add_db_item
    return await self.update_db_item(cur_item.item_id, album)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 189, in update_db_item
    "artist": json_serializer(ItemMapping.from_item(album_artist)),
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/media_items.py", line 260, in from_item
    return cls.from_dict(item.to_dict())
AttributeError: 'NoneType' object has no attribute 'to_dict'
```

**Comment by vk2him** [comment]
Also this one too:

```
Logger: music_assistant.mass
Source: /usr/local/lib/python3.9/site-packages/music_assistant/mass.py:242
First occurred: 9:28:13 AM (1 occurrences)
Last logged: 9:28:13 AM

Job [Library sync of artists for provider Filesystem] failed with error 'NoneType' object has no attribute 'provider'.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/__init__.py", line 390, in _library_items_sync
    db_item = await controller.get(
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/media_controller.py", line 68, in get
    return await self.add(details)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/artists.py", line 69, in add
    await self.match_artist(db_item)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/artists.py", line 83, in match_artist
    db_artist.provider == "database"
AttributeError: 'NoneType' object has no attribute 'provider'

```

**Comment by marcelveldt** [comment]
I've just published version 1.0.16 which contains a fix for this issue.
